pkgs/development/compilers/go/1.16.nix

```diff
@@ -45,7 +45,8 @@ let
     "riscv64" = "riscv64";
     "s390x" = "s390x";
     "powerpc64le" = "ppc64le";
-  }.${platform.parsed.cpu.name} or (throw "Unsupported system");
+    "mips64el" = "mips64le";
```

Comment:
This also needs to be added to 1.17 and 1.18.

Reply:
Thanks, fixed.